canonical-websites/jaas.ai
# Purge unused images

## Done

- Clean up codebase by removing unused images

## QA

- Pull feature branch
- Run `./run`
- Check there are no broken images in the site
